clusterloader2/pkg/util/util.go

```diff
@@ -111,6 +121,41 @@ func GetBoolOrDefault(dict map[string]interface{}, key string, defaultValue bool
 	return value, err
 }
 
+func getMap(dict map[string]interface{}, key string) (map[string]interface{}, error) {
```

Comment:
Do we need this? I don't think it's a part of the origin PR

Reply:
Good questions - this is what git suggested to me as a diff when I was cherry-picking d50a7c4983fe4f95e88ace79997ac39cf7958417. You are right, it's not needed. I will remove it.

However I'm not sure what are the best practices if we are doing a number of backports between master and release-1.18 - what is the best way to avoid conflicts.